src/site/content/en/blog/text-fragments/index.md

```diff
@@ -0,0 +1,423 @@
+</figure>
+
+### `prefix-` and `-suffix`
+
+Using long enough values for `textStart` and `textEnd` is one solution for obtaining a unique link.
+In some situations, however, this is not possible.
+On a side note, why did I choose the Chrome&nbsp;80 release blog post as my example?
+The answer is that in this release Text Fragments were introduced:
+
+<figure class="w-figure">
+  <img src="text-fragments.png" alt="Blog post text: Text URL Fragments.
+    Users or authors can now link to a specific portion of a page
+    using a text fragment provided in a URL.
+    When the page is loaded, the browser highlights the text and scrolls the fragment into view.
+    For example, the URL below loads a wiki page for 'Cat'
+    and scrolls to the content listed in the `text` parameter." class="w-screenshot" width="400">
+  <figcaption class="w-figcaption">Text Fragments announcement blog post excerpt.</figcaption>
+</figure>
+
+If I wanted to link to the word `text`
+that is written in a green code font at the bottom of the screenshot above,
+I would set `textStart` to `text`.
+Since the word `text` is, well, only one word, there cannot be a `textEnd`,
+but even just the blog post excerpt alone contains four times the word `text`.
+What now?
+The URL
+<a href="https://blog.chromium.org/2019/12/chrome-80-content-indexing-es-modules.html#:~:text=text"><code>https://blog.chromium.org/2019/12/chrome-80-content-indexing-es-modules.html<strong>#:~:text=text</strong></code></a>
+matches at the first occurrence of the word `Text`.
+
+{% Aside %}
+  Note that text fragment matching is case-insensitive.
+{% endAside %}
+
+<figure class="w-figure">
+  <img src="first-text.png" alt="" class="w-screenshot" width="400">
+  <figcaption class="w-figcaption">Text Fragment matching at the first occurrence of <code>Text</code>.</figcaption>
+</figure>
+
+Luckily there is a solution.
+In cases like this, I can specify a `prefix​-` and a `-suffix`.
+The word before the green code font "text" is "the", and the word after is "parameter".
+None of the other three occurrences of the word "text" has the same surrounding words.
+Armed with this knowledge, I can tweak the previous URL and add the `prefix-` and the `-suffix`.
+Like the other parameters, they, too, need to be percent-encoded.
+<a href="https://blog.chromium.org/2019/12/chrome-80-content-indexing-es-modules.html#:~:text=the-,text,-parameter"><code>https://blog.chromium.org/2019/12/chrome-80-content-indexing-es-modules.html<strong>#:~:text=the-,text,-parameter</strong></code></a>.
+To allow the parser to clearly identify the `prefix-` and the `-suffix`,
+they need to be separated from the `textStart` and the optional `textEnd` with a dash&nbsp;`-`.
+
+<figure class="w-figure">
+  <img src="correct-text.png" alt="" class="w-screenshot" width="400">
+  <figcaption class="w-figcaption">Text Fragment matching at the correct occurrence of <code>Text</code>.</figcaption>
+</figure>
```

Comment:
@robdodson Is there a way to make screenshots take the whole width. What he's illustrating is difficult to see. https://deploy-preview-2503--web-dev-staging.netlify.app/text-fragments/

Reply:
you can use `width="100%"` instead of `width="400"`

Reply:
I have made these three screenshots wider, since their text is important.